src/50_Sample_AMPs/Live_Blog.html

```diff
@@ -0,0 +1,123 @@
+      <script async src="https://cdn.ampproject.org/v0.js"></script>
+      <style amp-custom>
+        .carousel .slide > amp-img > img {
+        object-fit: contain;
+        }
+        .logo {
+          background-position: left 16px center;
+          background-repeat: no-repeat;
+          background-image: -webkit-image-set(
+            url(/img/ic_menu_white_1x_web_24dp.png) 1x,
+            url(/img/ic_menu_white_2x_web_24dp.png) 2x
+          );
+          background-color: #333;
+          text-align: left;
+          padding: 16px;
+          padding-left: 72px;
+          box-shadow: 0 2px 2px 0 rgba(0,0,0,.14),0 3px 1px -2px rgba(0,0,0,.2),0 1px 5px 0 rgba(0,0,0,.12);
+        }
+       .logo > a {
+         font-size: 16px;
+         font-weight: 500;
+         color: white;  
+         text-transform: uppercase;
+       }
+       .heading {
+         padding-bottom: 8px;
+       }
+       .heading > #summary {
+         font-weight: 500;
+       }
+       .heading > small {
+         color: #656565;
+       }
+
+      .date{
+          color: #a8a3ae;
+          font-size: 12px;
+       }
+      .blog {
+            padding: 0;
+            min-width: 70%;
+            background: #fff;
+      }
+      .blog p {
+            padding: 8px;
+            margin: 0;
+      }
+       .blog .text {
+            color: black;
+       }
+  </style>
+
+  </head>
+  <body>
+
+    <!-- -->
+    <div class="logo">
+      <a href="/"> AMP by Example</a>
+    </div>
+
+    <div class="heading">
+      <p><small class="date">18 July 2016</small></p>
+      <p id="summary">This is a sample article demonstrating how to write a live blog in AMP. It demonstrates the usage of amp-live-list component which allows to create live blogs.</p>
+    </div>
+
+    <amp-live-list
+    layout="container"
+    data-poll-interval="15000"
+    data-max-items-per-page="5"
+    id="amp-live-list-insert-blog">
+    <button id ="live-list-update-button" update on="tap:amp-live-list-insert-blog.update" class="button button-primary">You have updates</button>
+    <div items>
+      {{range .BlogItems}}
+          <div id="{{.Text}}" data-sort-time="{{.Timestamp}}">
+            <div class="card blog">
+              <a id="{{.Timestamp}}">
```

Comment:
you could assign the id to the `div` then you don't need to declare a separate anchor.